android/expoview/src/main/java/host/exp/exponent/notifications/channels/ScopedNotificationsChannelsFactory.java

```diff
@@ -0,0 +1,39 @@
+package host.exp.exponent.notifications.channels;
+
+import android.content.Context;
+
+import expo.modules.notifications.notifications.channels.AndroidXNotificationsChannelsFactory;
+import expo.modules.notifications.notifications.channels.managers.NotificationsChannelGroupManager;
+import expo.modules.notifications.notifications.channels.serializers.NotificationsChannelGroupSerializer;
+import expo.modules.notifications.notifications.channels.managers.NotificationsChannelManager;
+import expo.modules.notifications.notifications.channels.serializers.NotificationsChannelSerializer;
+import host.exp.exponent.kernel.ExperienceId;
+
+public class ScopedNotificationsChannelsFactory extends AndroidXNotificationsChannelsFactory {
+  private ExperienceId mExperienceId;
+
+  public ScopedNotificationsChannelsFactory(Context context, ExperienceId experienceId) {
```

Comment:
Maybe we could rename it to Provider and provide always the same instance, lazily created?

Reply:
done